README.md

```diff
@@ -34,6 +34,10 @@ jobs:
           W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
           # Replace following with appropriate value. See options.md for details.
           W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-group/2014JulSep/1234.html
+          # Usually, you want the following set too...
+          W3C_BUILD_OVERRIDE: |
+             shortName: your-specs-shortname-here
+             specStatus: WD
```

Comment:
I would also be super happy to see the `W3C_BUILD_OVERRIDE` example added — but since `specStatus` is respec-only, it would also be great if (somehow) and example of the bikeshed equivalent, `status`, could also be added.

Or, stepping back, maybe it would be even greater if in respec, `status` were added as an alias for `specStatus` — then the example could just use `status`. Or bikeshed could add `specStatus` as an alias for its `status`.

Reply:
Good suggestion.... we could even support status in ReSpec itself.

Reply:
Probably shortname also. ReSpec has shortName.

Or maybe add a mapping in spec-prod? Avoids multiple ways to do things ReSpec, which is less confusing for users.

Reply:
yeah, mapping might be better. Should we do that as a followup?

Reply:
> Or maybe add a mapping in spec-prod? Avoids multiple ways to do things ReSpec, which is less confusing for users.

Yeah having spec-prod abstract away the difference would be better for users. So do you mean that spec-prod users could give either `shortName` or `shortname` and `specStatus` or `status` — and it will all just word, regardless of whether they’re using ReSpec or Bikeshed?

Reply:
I was thinking of supporting one of them, but supporting both is better!

Reply:
> yeah, mapping might be better. Should we do that as a followup?

Yeah — seems like this can be merged first